drivers/hmis/config/initializers/hmis_feature.rb

```diff
@@ -15,3 +15,8 @@
 Rails.application.config.queued_tasks[:hmis_check_constraints] = -> do
   Hmis::Tasks::CheckConstraints.check_hud_constraints
 end
+
+Rails.application.config.queued_tasks[:hmis_7266_data_cleanup] = -> do
```

Comment:
This seems fine, though I'm not sure I'd use an internal ID in the key.  Can we just call it:
```suggestion
Rails.application.config.queued_tasks[:hmis_disabling_condition_and_race_cleanup] = -> do
```

That said, it does tag it to a particular point in time, so maybe it should just be a rough date:
```
Rails.application.config.queued_tasks[:hmis_disabling_condition_and_race_cleanup_2_2025] = -> do
```

Reply:
Thanks! Updated to your second suggestion